tests/setup.js

```diff
@@ -9,7 +9,7 @@ import '@testing-library/jest-dom';
 global.$ = $;
 global.jQuery = $;
 global.performance = {};
-global.fetch = jest.fn().mockResolvedValue({status: 200});
+require('isomorphic-fetch');
```

Comment:
Previously, we couldn't use fetch in webapp tests. Now we can!

Also, it might make sense to swap this for `node-fetch` in the future since that's what `isomorphic-fetch` uses under the hood for non-browser environments.